### PR TITLE
/status

### PR DESCRIPTION
**Pull request by BIRD311**

### CODE REVIEW

File: MomentumDiscordBot/Commands/Moderator/ModeratorModule.cs [hunk 1]
```diff
@@ -47,5 +49,20 @@ public async Task GetMembersWithRoleAsync(InteractionContext context, [Option("r
...
             }
         }
+
+        [SlashCommand("status", "Sets the bots status")]
+        public async Task StatusAsync(InteractionContext context,
+            [Option("status", "status")] string status,
+            [ChoiceProvider(typeof(ActivityTypeChoiceProvider))][Option("type", "ActivityType")] string type = null)
+        {
+            DiscordActivity activity;
+            if (Enum.TryParse(type, out ActivityType activityType))
```
Comment (tsa96) [inline MomentumDiscordBot/Commands/Moderator/ModeratorModule.cs]: This seems like a reasonable place for a ternary i.e.
```c#
activity = Enum.TryParse(type, out ActivityType activityType) ? new DiscordActivity(status, activityType) : new DiscordActivity(status);
```

Comment (tsa96) [reply MomentumDiscordBot/Commands/Moderator/ModeratorModule.cs]: In fact you can then just merge with the above line w/ 
```c#
 var activity = Enum.TryParse(type, out ActivityType activityType) ? new DiscordActivity(status, activityType) : new DiscordActivity(status);
```

**Review by tsa96 — CHANGES_REQUESTED** [review]
Just a small code thing.

It's annoying Discord's API for this is so shitty. Why would they give bots the ability to set what game they're "playing" or whatever (which is literally never actually the case, it's a bot ffs), but not set a regular custom status? Anyway this looks like the best we can really do, just not a very useful feature given the limitations.

File: MomentumDiscordBot/Commands/Moderator/ModeratorModule.cs (hunk 1 above)
Comment (BIRD311) [reply MomentumDiscordBot/Commands/Moderator/ModeratorModule.cs]: I was thinking about using something like this but I think it gets hard to read really quick. But with newlines I like it. Hope you don't mind.

Comment (tsa96) [reply MomentumDiscordBot/Commands/Moderator/ModeratorModule.cs]: Oh sure, newlines in this case definitely more readable. I'm too used to JS where I use ternaries absolutely everywhere and get very used to reading them lol

**Review by tsa96 — APPROVED** [review]

**Pull request merged**